src/dtos/update-student-profile-info.dto.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  IsArray,
+  IsEmail,
+  IsEnum,
+  IsInt,
+  IsNotEmpty,
+  IsNumber,
+  IsOptional,
+  IsString,
+  Min,
+} from 'class-validator';
+import {
+  ExpectedContractType,
+  ExpectedWorkType,
+  StudentStatus,
+} from '../entities/students-entity';
+
+export class UpdateStudentProfileInfoDto {
+  @IsEmail()
+  @IsNotEmpty()
+  email: string;
+
+  @IsOptional()
+  @IsNumber()
+  @Min(9)
+  tel: string;
+
+  @IsNotEmpty()
+  @IsString()
+  githubUsername: string;
+
+  @IsNotEmpty()
+  @IsString()
+  firstName: string;
+
+  @IsNotEmpty()
+  @IsString()
+  lastName: string;
+
+  @IsArray()
+  portfolioUrls: string[];
+
+  @IsNotEmpty()
+  @IsArray()
+  bonusProjectUrls: string[];
```

Comment:
To jest jednak osobna kolumna: linki do repozytoriów projektu indywidualnego, a admin/ka podaje linki do repozytoriów projektu grupowego. Dlatego różne nazwy.
```suggestion
  projectUrls: string[];
```

Reply:
Mają być trzy osobne kolumny:

1. _bonusProjectUrls_ z linkami do projektu grupowego, którą wypełnia admin/ka w pliku CSV,
2. _projectUrls_ z linkami do projektu indywidualnego, którą wypełnia kursant/ka, obowiązkowe,
3. _portfolioUrls_ z linkami do czego tam sobie kursant/ka chce, np. do stronki w Wordpressie, którą chce się pochwalić, wypełnia kursant/ka, nieobowiązkowe